analysis/07b_an_multivariable_cox_models_FULL_sense_11thJuneCensor.do

```diff
@@ -78,7 +78,7 @@ local outcome `1'
 
```

Comment:
I believe we dropped this analysis in the end, because we decided that although the football we a big event, there'd be others causing people to gather and we couldn't account for all events.

Reply:
Sure!